scania-digital-design-system/sdds
# AB#292 Feature/textfield component

**Describe pull-request**  
_Describe what the pull-request is about_

- Added textfield component
- Added variables for textfield component
- Added story for textfield
- Improved build time, for netlify! Can be improved even more

Storybook settings
- Added story for divider
- Added background, viewport  and grid settings for storybook

- Remove some files

Next step:

TODO:
- Helper text
- Suffix
- Prefix
- Error
- Textcounter

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: [AB#292](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/292)

**How to test**  
_Add description how to test if possible_
- Run storybook/netlify
- Possible to use regular html for some of the component parts
- Label inside needs to be a web-component
- Check textfields


**Additional context**  
_Add any other context about the pull-request here._
is it okey to supply regular html AND web-component?
